app/helpers/fhir_helper.rb

```diff
@@ -242,22 +242,22 @@ def close_contact_from_fhir(related_person)
   end
 
   def history_as_fhir(history)
+    history_creator_reference = FHIR::Reference.new
+    history_creator_reference.identifier = history.created_by
```

Comment:
`Identifier` is a type in FHIR, so it itself cannot have a value. This should be:
```ruby
history_creator_reference.identifier = FHIR::Identifier.new(value: history.created_by)
```
In fact I think you can just do this all in one line:
```ruby
          who: FHIR::Reference.new(identifier: FHIR::Identifier.new(value: history.created_by)),
```
If `history.created_by` was `nil` (which I don't think should happen anyway), the gem that serializes this to JSON will ignore that value.